subjects/java/checkpoints/time-tracker/README.md

```diff
@@ -0,0 +1,69 @@
+## TimeTracker
+
+### Instructions
+
+Write a `ProjectTime` class to track the duration of a project using start and end times. You will be asked to add getter and setter methods for the `startTime` and `endTime` attributes. When you set one of them, you must update the `hoursLogged` attribute. The `hoursLogged` getter should output the hours in the following format:
+
+- Less than 120 minutes: `hoursLogged` m
+- Less than 120 hours: `hoursLogged` h
+- Less than 120 days: `hoursLogged` d
+- More than 120 days: `hoursLogged` mo
+
+If there is any error, the number of `hoursLogged` should be handled accordingly.
+
+> 💡 The `hoursLogged` shouldn't be a negative number except -1 in case of errors.
+
+### Expected Functions
+
+```java
```

Comment:
Consider adding the imports used in the solution here 01-edu/java-tests#68 to make it clear that it's possible to use external libraries

Reply:
Yeah you are right